src/AssetCompiler.php

```diff
@@ -60,16 +60,14 @@ public function __construct(FilterRegistry $filters, $debug)
      */
     public function generate(AssetTarget $build)
     {
-        $filters = $this->filterRegistry->collection($build);
+        $filters = $this->filterRegistry->collection($build, $this->debug);
         $output = '';
         foreach ($build->files() as $file) {
             $content = $file->contents();
             $content = $filters->input($file->path(), $content);
             $output .= $content . "\n";
         }
-        if (!$this->debug || php_sapi_name() === 'cli') {
-            $output = $filters->output($build->path(), $output);
-        }
+        $output = $filters->output($build->path(), $output);
```

Comment:
This doesn't apply minifiers all the time. Inside individual postprocessor filter's `output` method, it checks whether to apply filter or not. Normally it won't apply. If `autoDebugMode` option is set to value other than zero, it will apply in debug mode as well. I thought this could be useful in some cases such as postcss' autoprefixer.
And sometimes, for debugging purpose, if we set `debug` to `false` in `app.php` and errors occur during applying postprocessors, we can't figure it out as it just says `Internal Error Occured`. In that case, we can set `debug` to `true` and check what errors and where they come from.